### PR TITLE
Update timely response percentage

### PR DESCRIPTION
**Pull request by niqjohnson**
The percent of complaints that receive timely responses has gone up to 98%.

## Changes

- Bumped the percent of timely responses shown on `/data-research/consumer-complaints/` to 98%
- Fixed the missing bottom margin on the timely response callout at small screen sizes

## How to test this PR

1. Go to `/data-research/consumer-complaints/`
2. Confirm the number in the "complaints sent to companies get timely responses" callout is "98%" at all screen sizes (the content appears in the markup twice, once for large screens and once for small screens 😢 )
3. Confirm that callout has a 15px bottom margin at small screen sizes

## Screenshots

Large screens | Small screens
---- | ----
large-screens | small-screens

## Checklist

- [x] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [x] Changes are limited to a single goal (no scope creep)
- [x] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)

### CODE REVIEW

**Review by Scotchester — APPROVED** [review]
Why is this still on hold?

**Comment by niqjohnson** [comment]
@Scotchester: Still waiting for final stakeholder sign-off.

**Pull request merged**
